integration/client/client_test.go

```diff
@@ -546,3 +548,50 @@ func TestDefaultRuntimeWithNamespaceLabels(t *testing.T) {
 		t.Error("failed to set default runtime from namespace labels")
 	}
 }
+
+func TestRuntimeInfo(t *testing.T) {
+	t.Parallel()
+
+	if runtime.GOOS == "windows" {
+		t.Skip("io.containerd.runhcs.v1 does not implement `containerd-shim-runhcs-v1.exe -info` yet")
+	}
+
+	client, err := newClient(t, address)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+
+	ctx, cancel := testContext(t)
+	defer cancel()
+
+	rti, err := client.RuntimeInfo(ctx, "", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if rti.Name == "" {
+		t.Fatal("got empty RuntimeInfo.Name")
+	}
+
+	if os.Getenv("RUNC_FLAVOR") == "crun" {
+		t.Skip("crun does not implement `crun features` yet: https://github.com/containers/crun/issues/1177")
```

Comment:
It is supported since 1.8.6, maybe it is better to update crun here and don't skip the test?

Reply:
1.8.7 for a bugfix regarding features, sorry, not 1.8.6

Reply:
Fixed